.gitignore

```diff
@@ -1,6 +1,10 @@
 # Setuptools SCM version.py file
 pydecorate/version.py
 
+# Files created by test run
+style_retention.png
+add_logo_pytroll.png
```

Comment:
Is the `style_retention.png` one created by your new test too?

The test should be updated to use `tmp_path` for where it writes files.

Reply:
And for logo one I guess I could check that the new file is not all black for an indication it worked.

Reply:
If you're not familiar, you could use the tmp_path pytest fixture: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
And yeah checking for a non-black image sounds good enough.